<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14px Poppins;</string>
  </property>
  <widget class="QLabel" name="lblOpacity">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="Widget1" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: white;</string>
   </property>
   <widget class="QLineEdit" name="periodLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 14px &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-size: 14px;
	font-family: Poppins;
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-size: 14px;
	font-family: Poppins;
	font-weight: 500;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="captureCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>95</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 10px;
	font: 14px &quot;Poppins&quot;;
}

QCheckBox::indicator {
    width: 20px; /* Ancho del indicador */
    height: 20px; /* Alto del indicador */
    border-radius: 4px; /* Bordes redondeados */
    border: 2px solid #666; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: #00007f; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>Activar captura automática</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalSamplesLine">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: solid;
border-width: 1px;
border-radius: 6px;
font: 14px &quot;Poppins&quot;;
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Opciones de captura</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>65</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de muestras</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>145</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Período (s):</string>
    </property>
   </widget>
   <zorder>periodLine</zorder>
   <zorder>closeBtn</zorder>
   <zorder>captureCheckBox</zorder>
   <zorder>totalSamplesLine</zorder>
   <zorder>confirmBtn</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
  <widget class="QWidget" name="widgetKeyboard" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>202</y>
     <width>480</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 234, 234);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
